### DISCUSSION TITLE
msidentity-app-sync tool

### DISCUSSION THREAD

**Discussion by jennyf19** [AzureAD/microsoft-identity-web#1067] [Show and tell]
Have you tried the new [msidentity-app-sync tool](https://www.nuget.org/packages/msidentity-app-sync/)? If so, let us know your thoughts and feedback!

https://www.youtube.com/watch?v=YCORmpTbt6g
[More info in the README](https://github.com/AzureAD/microsoft-identity-web/blob/master/tools/app-provisioning-tool/README.md)
and experience in [VS](https://github.com/AzureAD/microsoft-identity-web/blob/master/tools/app-provisioning-tool/vs2019-16.9-how-to-use.md).

🐈

**Comment by jstynder** [comment]
Would be great if the tool can generate scripts, or instructions for the updates/changes. This will allow a hand-over to a different team that handles deployment

**Comment by jmprieur** [comment]
yes, that's an awesome idea (which we were thinking of. Thanks for confirming!)
